### PR TITLE
Ana/hide currency selector for terra

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

Easy and effective!

Checks if the staking denom has fiatValue and in only in that case displays the currency 
selector.

Thank you! 🚀

---

For contributor:

- [x] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/components/common/TmBalance.vue [hunk 1]
```diff
@@ -19,7 +19,10 @@
                 {{ overview.totalStake | bigFigureOrShortDecimals | noBlanks }}
               </h2>
             </div>
-            <div v-if="isMultiDenomNetwork" class="currency-selector">
+            <div
+              v-if="isMultiDenomNetwork && stakingBalance.fiatValue"
```
Comment (faboweb) [inline src/components/common/TmBalance.vue]: HACK

Comment (Bitcoinera) [reply src/components/common/TmBalance.vue]: it actually makes a lot of sense, I think.

But I guess you could call every piece of software ever written in human History a "hack"

Comment (faboweb) [reply src/components/common/TmBalance.vue]: äh no ;P

Comment (Bitcoinera) [reply src/components/common/TmBalance.vue]: xD

**Pull request merged**